apisix/init.lua

```diff
@@ -82,6 +75,14 @@ end
 
 
 function _M.http_init_worker()
+    local seed, err = core.utils.get_seed_from_urandom()
+    if not seed then
+        core.log.warn('failed to get seed from urandom: ', err)
+        seed = ngx_now() * 1000 + ngx.worker.pid()
+    end
+    math.randomseed(seed)
+    core.log.info("random test in [1, 10000]: ", math.random(1, 1000000))
```

Comment:
should be removed

Reply:
why do we need to remove this line?
It is necessary for debugging.

Reply:
Can it help user to find any problems?

Reply:
it not for the user, the developer need it for testing.

here is the code link of the test case: https://github.com/apache/apisix/pull/2357/files#diff-a46ca98b2569d841ad4742ef90427a1fR39

Reply:
@membphis 
I think we should add a comment for it.